doc/samples/sample_notify_fifo.sh

```diff
@@ -24,6 +24,7 @@ FIFO=$1
 [[ -z $FIFO ]] && echo "A FIFO name must be specified" && exit 1
 
 LOG_FILE=/tmp/${FIFO##*/}.log
+PID_FILE=/var/run/${FIFO##*/}.pid
 
```

Comment:
> although I am not sure that [[ -n $FD ]] && exec {FD}>&- is needed since the script is exiting.

The file descriptor is always removed when the script exits.
> `trap exiting EXIT`

Does signal EXIT exist ? I am not sure

Reply:
> The file descriptor is always removed when the script exits.

Are you confirming that `[[ -n $FD ]] && exec {FD}>&-` is not needed?

In addition to the real signals, the valid sigspecs for trap also include EXIT, DEBUG, RETURN and ERR. As the bash man page says:
> If a sigspec is EXIT (0) the command arg is executed on exit from the shell.

Reply:
> Are you confirming that [[ -n $FD ]] && exec {FD}>&- is not needed?

Not needed